spec/requests/posts_spec.rb

```diff
@@ -13,13 +15,15 @@
     end
     it 'has the correct text placeholderss' do
       body = response.body
-      expect(body).to include('Posts#index')
+      expect(body).to include(@post.title)
     end
   end
 
   describe 'GET /show' do
     before :each do
-      get '/users/1/posts/1'
```

Comment:
- [ ] Finally, we shall once more check the bullets point on the screenshot below and write unit tests that assert the different scenarios as stipulated in your [project's requirements.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md)

![Screenshot from 2022-09-06 02-22-34](https://user-images.githubusercontent.com/63087491/188527487-6fdddd9f-3d3b-4c8e-accd-d99043e53826.png)

Reply:
the tests are inside the folder views, these are old tests for the requests

Reply:
@brahimdidi, you are correct for any single of your comments. I just discovered my error rights away. I ended up running **rspec** on the console after setting up your project in my local environment. Usually, this commands using runs through all the test files in the /spec directory. After discovering that you had less than 35 test cases, I couldn't pay attention that much to see if some files were in the spec/views directory.

I'm sorry for that error.